services/cinema_hall.py

```diff
@@ -7,9 +7,16 @@ def get_cinema_halls() -> QuerySet:
     return CinemaHall.objects.all()
 
 
-def create_cinema_hall(
-    hall_name: str, hall_rows: int, hall_seats_in_row: int
-) -> CinemaHall:
-    return CinemaHall.objects.create(
-        name=hall_name, rows=hall_rows, seats_in_row=hall_seats_in_row
-    )
+def create_cinema_hall(hall_name: str,
+                       hall_rows: int,
+                       hall_seats_in_row: int) -> CinemaHall:
+    return CinemaHall.objects.create(name=hall_name,
+                                     rows=hall_rows,
+                                     seats_in_row=hall_seats_in_row)
+
+
+def get_cinema_hall_by_id(cinema_hall_id: int) -> CinemaHall | None:
```

Comment:
There was no requirement to make this but even if you had to, why would you return None if cinema hall doesn't exist? Shouldn't warning be more appropriate here?

Reply:
I don’t remember why I did it, but what I did it for was not included in the solution, so it was redundant, I deleted it.